### PR TITLE
Changed the order of the tabs in the navbar, fixed logo ratio, and ad…

### PR DESCRIPTION
**Pull request by JFPach98**
### What does this PR do?

- Changed the order of the tabs in the navbar, fixed logo ratio, and added sponsor to navbar.

#### Related issues

#77 

#### Merge Checklist:

- [x] The branch is up-to-date (i.e. rebased) with the base branch
- [x] No code style issues

### CODE REVIEW

**Review by AuroTB — CHANGES_REQUESTED** [review]
I've added a new image called _steren2.png_ to the sponsor's folder, so you can use it for the logo.
Also, please fix the responsive view.
image

**Comment by JFPach98** [comment]
> I've added a new image called _steren2.png_ to the sponsor's folder, so you can use it for the logo.
In which branch?

File: src/components/NavBar/NavBar.css [hunk 1]
```diff
@@ -62,3 +63,35 @@
...
+#app-navbar #navbar-legend-div1 {
+  color: #888B8D;
+  height: 48px;
+  vertical-align: middle;
+  display: inline-block;
+  position: absolute;
+  right: 2px;
+}
+
+@media all and (max-width: 991px){
```
Comment (AuroTB) [inline src/components/NavBar/NavBar.css:86]: #navbar-legend-div1 has almost the same styling as #navbar-legend-div, why don't you try using the same styling for both of them? and instead, use `@media all and (min-width)` to change position and right.

File: src/components/NavBar/NavBar.js [hunk 2]
```diff
@@ -120,11 +122,46 @@ class NavBar extends Component<Props, State> {
...
+                  Powered by
+                </p>
+                
+              </div>
+              
+            </Navbar.Brand>
 
           </Nav>
         </Navbar.Collapse>
+        <Navbar.Brand id="navbar-legend-div1">
```
Comment (AuroTB) [inline src/components/NavBar/NavBar.js:151]: Nice!

**Review by AuroTB — CHANGES_REQUESTED** [review]
Looks nice! I've just added some comments.

**Review by AuroTB — APPROVED** [review]
Everything looks great! Good job.
Before merging please resolve all comments and you are good to go!

**Pull request merged**